server/service/src/settings_service.rs

```diff
@@ -87,11 +87,11 @@ pub trait SettingsServiceTrait: Sync + Send {
                     Some(settings.central_server_site_id as i32),
                 )?;
                 key_value_store.set_i32(
-                    KeyValueType::SettingsSyncSideId,
+                    KeyValueType::SettingsSyncSiteId,
                     Some(settings.site_id as i32),
                 )?;
                 key_value_store.set_string(
-                    KeyValueType::SettingsSyncSideHardwareId,
+                    KeyValueType::SettingsSyncSiteHardwareId,
                     Some(settings.site_hardware_id.clone()),
```

Comment:
hmm we should make an issue, site_hardware_id should be mac address or some other unique identifier (we should get that pragmatically and also include in some sync API headers)

Reply:
it's mentioned here: https://github.com/openmsupply/open-msupply/issues/21